src/script/utils/analytics.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * const exampleValues = {
+      isAuto: false,
+      behavior: 0,
+      uri: window.location.href,
+      pageName: "featuresPage",
+      pageHeight: window.innerHeight
+    };
+ */
+
+export interface AnalyticsOptions {
+  isAuto?: boolean,
+  behavior?: number,
+  uri?: string,
+  pageName?: string,
+  pageHeight?: number
+}
+
+export function capturePageView(options: AnalyticsOptions) {
+  (window as any).awa.ct.capturePageView(options);
```

Comment:
Because awa will not exist if a user is running an ad blocker, can we check for its existence before use?